_server.js

```diff
@@ -0,0 +1,158 @@
+const electron = require('electron'),
+app = electron.app,
+BrowserWindow = electron.BrowserWindow,
+ipcMain = electron.ipcMain
```

Comment:
nits: semicolons at the end of each statement; indent subsequent variable declaration, here and below:

```js
const foo = 1,
  bar = 2,
  baz = 3;
```